LifeCycleAssessment_Engine/Query/EvaluateElement.cs

```diff
@@ -0,0 +1,177 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using BH.oM.Base.Attributes;
+using BH.oM.Dimensional;
+using BH.oM.LifeCycleAssessment.MaterialFragments;
+using BH.oM.LifeCycleAssessment.Results;
+using BH.oM.LifeCycleAssessment;
+using BH.oM.Physical.Materials;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using BH.Engine.Matter;
+using BH.Engine.Spatial;
+
+namespace BH.Engine.LifeCycleAssessment
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+
```

Comment:
We should retain the existing Evaluate Reference Value method within LCA toolkit until the proposed "EvaluateElement" method is able to evaluate non-IElementM objects

Reply:
Happy to keep it for now. Have not touched that code at all, so should still be present.

THere are scope to do some restructuring of the Toolkit imo, but did not want to do that in this PR, which is just meant to streamline and hopefully improve this new workflow.